app/javascript/packs/pages/browse_requests/VolunteerInstructions.jsx

```diff
@@ -1,102 +1,205 @@
+              <NavLink className={classes.volunteerTitleLink} to={SIGNUP_PATH}>
+                Offer Help Now
+              </NavLink>
+            </Typography>
+          </Grid>
+          <Hidden smDown>
+            <Grid item md={6}>
+              <DoctorAndVolunteer />
+            </Grid>
+          </Hidden>
+        </Grid>
+        <Box mb={4}>
+          <Typography className={classes.volunteerInfoText} variant="body1">
+            As a volunteer or a business donor, you will provide essential
+            support through hands-on help, outreach, or donations of products or
+            services. You can offer to help in a variety of ways, including:
+          </Typography>
+        </Box>
+        <Grid container>
+          {offerTypes.map(({ OfferComponent, offerText }) => (
+            <Grid
+              key={offerText.replace(/\s/g, "_")}
+              item
+              className={classes.offerGridItem}
+              xs
+            >
+              <OfferComponent />
+              <Typography>{offerText}</Typography>
+            </Grid>
+          ))}
+        </Grid>
+        <Hidden smDown>
+          <Box mt={6}>
+            <Typography className={classes.volunteerInfoText} variant="body1">
+              Our volunteer coordinators will match you with healthcare workers
+              in your area who need help.
+            </Typography>
+          </Box>
+        </Hidden>
+        <Box className={classes.alignCenter} mt={4} mb={4}>
           <Button
-            className={classes.volunteerButton}
-            size="large"
+            className={classes.button}
             variant="contained"
             elevation={0}
+            href={SIGNUP_PATH}
           >
-            <div className={classes.volunteerButtonText}>Sign Up Now</div>
+            <div className={classes.buttonText}>Offer Help</div>
           </Button>
-        </ReactLink>
-        <Typography className={classes.share}>Share</Typography>
-        <Container maxWidth="xs">
-          <Grid container justify="center">
-            <Grid container item xs={3} sm={2} justify="center">
-              <Link href={socialMediaLinks.FACEBOOK_LINK}>
-                <FacebookIcon />
-              </Link>
-            </Grid>
-            <Grid container item xs={3} sm={2} justify="center">
-              <Link href={socialMediaLinks.TWITTER_LINK}>
-                <TwitterIcon />
-              </Link>
-            </Grid>
-            <Grid container item xs={3} sm={2} justify="center">
-              <Link href={socialMediaLinks.LINKEDIN_LINK}>
-                <LinkedInIcon />
-              </Link>
-            </Grid>
-            <Grid container item xs={3} sm={2} justify="center">
-              <Link href={socialMediaLinks.EMAIL_LINK}>
-                <EmailIcon />
-              </Link>
-            </Grid>
+        </Box>
+      </Container>
+      {/* Map component goes here */}
```

Comment:
I used two containers here because I noticed the map component exceeded the bounds of the container, and I've run into some headaches in the past wrangling components who want to do that. If it's not an issue, it will be a simple matter to remove the wrapping Fragment, and make everything go in one Container again.

Reply:
This is great! The map component defaults to full width with aspect ratio enforced by  padding bottom, so it should fit right in here!